Makefile

```diff
@@ -1,11 +1,11 @@
 NAME      := mpifileutils
 SRC_EXT   := gz
 REPO_NAME := mpifileutils-pkg
-GIT_SHORT := bdc379b
+#GIT_SHORT := bdc379b
 
-BUILD_DEFINES := --define "git_short $(GIT_SHORT)"
+#BUILD_DEFINES := --define "git_short $(GIT_SHORT)"
 
-RPM_BUILD_OPTIONS := $(BUILD_DEFINES)
+#RPM_BUILD_OPTIONS := $(BUILD_DEFINES)
```

Comment:
I kept this since we will very likely build with a patch again in the future.

Reply:
These should all be removed as they are obsolete now due to a formalization of this kind of git patch handling rather than each package re-inventing it.  `PKG_GIT_COMMIT` and `GITHUB_PROJECT` or `GIT_PROJECT` should be used instead.  An example can be found in the ior package's [Makefile](https://github.com/daos-stack/ior/commit/ee349bd638297a75c5affe7c003971bdf4fdfd41).

You will then need to convert the uses of `%{git_short}` to `%{commit}` such as [ior.spec](https://github.com/daos-stack/ior/blob/master/ior.spec) exemplifies.